include/antidote.hrl

```diff
@@ -50,7 +52,7 @@
 %% wake up and retry. I.e. a read waiting for
 %% a transaction currently in the prepare state that is blocking
 %% that read.
--define(SPIN_WAIT, 10).
+-define(SPIN_WAIT, 0).
```

Comment:
that's how much time cure waits when there is a transaction being prepared. 
I got, by running benchmarks, that throughput and latency increase when setting it to 0.
should I put it back to 10 anyway?